wallet/api/admin_update_network.go

```diff
@@ -10,8 +10,9 @@ import (
 )
 
 type AdminUpdateNetworkParams struct {
-	Name string `json:"name"`
-	API  struct {
+	Name     string             `json:"name"`
+	Metadata []network.Metadata `json:"metadata"`
+	API      struct {
```

Comment:
beyond the scope of this PR, but IMO it's nicer/cleaner to not nest types like this